### PR TITLE
fix: nan CTC loss

### PR DESCRIPTION
**Pull request by cannguyen275**
Hi @kwon-evan,
Fix nan_loss in my trained process by:
- With image size [100, 50], number of timestep is 19. Then I have changed `t_length` from 18 to 19
- Missing bank character for idn_config, so I have added `-` to `chars`
- [This practice](https://discuss.pytorch.org/t/best-practices-to-solve-nan-ctc-loss/151913) shows `gradient_clip_val` might help to reduce nan CTC loss. I'm not sure if it is solution for nan loss issue, but it's nice to have.

### CODE REVIEW

**Comment by cannguyen275** [comment]
@kwon-evan I've finally achieved 97% of accuracy (from 93% - brown line in below graph) by:
- Generate 1M synthesize data, use [LPRNet Tensorflow](https://github.com/bluesy7585/tensorflow_LPRnet/blob/master/gen_plates.py) version.
- Train a pre-trained model using 1M data (green line in below graph). Then train a final model by using the pre-trained mode with real data and random synthesize data (25% randomly - pink line in below graph).

Do you think it could make any senses to add those flow into this project?

Screenshot 2023-11-16 at 20 17 14

**Review by kwon-evan — APPROVED** [review]
Thanks for contribute!

**Comment by kwon-evan** [comment]
> Do you think it could make any senses to add those flow into this project?

I think it's good to have the model pre-trained with synthesize data in repo.
Any contribution is welcome.
